### PR TITLE
Add search bar

### LINKED ISSUE #157: Feature Request: Find manager/curator
**Is your feature request related to a problem?**
It would be cool if you could search for a certain manager or curator, because it's a big list to search through.
A small bit of code I've gotten that may help is `document.querySelectorAll("h4")[0].childNodes[0].innerText`.
This could be made with a form element, or separate \<input> and \<button> elements

**Describe alternatives you've considered**
Um, I feel like a search would be the best, I can't think of alternatives

**Additional context**
None

I'm currently working on something for this.

### PR DESCRIPTION
**Pull request by Accio1**
### Resolves:

Resolves #157 
Thanks to @gosoccerboy5 for suggesting this feature!

### Changes:

-Adds a search bar to the List of Forum Helpers page.
-The search bar border turns red if there is an invalid username and green if the username is valid.
-The search button can only be clicked if the username is valid.
-When the search button is clicked, it will direct the page to that person's username on the page.

### Local Tests:

Tested locally in both desktop and mobile views.

@gosoccerboy5 I decided to use IDs instead of classnames as you had proposed just because it seemed to make the code a bit more condense. If you have any other features you would like, feel free to suggest them below create a PR to merge into this branch with your changes.

@leahcimto

### CODE REVIEW

File: forumhelpers/index.html [hunk 1]
```diff
@@ -28,6 +28,13 @@ <h3 class="forumhelpers_header">Below is a list of current Forum Helpers, along
 		<br>
 		<p class="forumhelpers_header">There are currently <span id="managersNumber"></span> Managers and <span id="curatorsNumber"></span> Curators in our studio, for a total of <span id="totalNumber"></span> members.</p>
 		<!--<h4 style="margin-left: 10px;"><img src="/resources/coder.png" alt="coder" style="width:30px;height:30px;vertical-align:middle"> Indicates that that person has helped code the website.</h4>-->
+
+		<div class="searchBlock">
+			<p><strong>Search for a Forum Helper</strong></p>
```
Comment (leahcimto) [inline forumhelpers/index.html:33]: Could this possibly be a bit bigger?

Comment (Accio1) [reply forumhelpers/index.html]: Done, I changed it to a `<h3>` element instead of `<p>`, which increases the font size.

Comment (leahcimto) [reply forumhelpers/index.html]: Great!

**Review by leahcimto — APPROVED** [review]

**Comment by gosoccerboy5** [comment]
You guys fine with this? No bugs?

**Review by gosoccerboy5 — APPROVED** [review]
Wow, looks awesome! You really added some good stuff

File: forumhelpers/FHP.js [hunk 2]
```diff
@@ -63,3 +63,38 @@ function getCount(name) {
...
+	else {
+		searchBar.style.borderColor = "black"
+		searchButton.style.borderColor = "black";
+		searchButton.disabled = true;
+		searchButton.title = "Invald Username";
+	}
+}
+
+function searchUser() {
+	console.log("Hi")
```
Comment (gosoccerboy5) [inline forumhelpers/FHP.js:95]: Um.. ok

Comment (Accio1) [reply forumhelpers/FHP.js]: Whoops! Forgot to remove that.

File: forumhelpers/FHP.js [hunk 3]
```diff
@@ -63,3 +63,37 @@ function getCount(name) {
...
+		searchBar.style.borderColor = "red";
+		searchButton.style.borderColor = "red";
+		searchButton.disabled = true;
+		searchButton.title = "Invald Username";
+	}
+	else {
+		searchBar.style.borderColor = "black"
+		searchButton.style.borderColor = "black";
+		searchButton.disabled = true;
+		searchButton.title = "Invald Username";
```
Comment (gosoccerboy5) [inline forumhelpers/FHP.js:90]: *invalid (also on line 84)

Comment (gosoccerboy5) [reply forumhelpers/FHP.js:90]: By the way, is it possible to clear out the tooltip when the user clears the search bar?
Screen Shot 2021-05-15 at 2 16 16 PM

Comment (Accio1) [reply forumhelpers/FHP.js]: Yes, that is possible. I left the tooltip because the bar being empty would be an invalid username.

Comment (gosoccerboy5) [reply forumhelpers/FHP.js]: ig that makes sense

**Comment by Accio1** [comment]
Alright, I'm going to go ahead and merge now!

**Comment by gosoccerboy5** [comment]
Oh shoot! Would it be possible to add support for the enter/return key, or is it too late for that? It looks like the branch is deleted

**Pull request merged**